code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -82,9 +105,28 @@
 			spawn(det_time)
 				prime()
 
+
+/obj/item/weapon/grenade/iedcasing/proc/AddShrapnel(var/obj/item/I, mob/user as mob)
```

Comment:
You should really use snake_case to be consistent with the rest of the file and most of the codebase.